docs/src/pages/guides/DB2OnCloudSetup.md

```diff
@@ -167,9 +167,45 @@ The FHIR server uses a proxy datasource mechanism, allowing new datasources to b
         }
 ```
 
-Note that no username properties are given, because the authentication module only requires the API-KEY.
+#### Mapping from IBM Db2 on Cloud Endpoint Credentials
 
+This section explains how to populate the FHIR datasource from IBM Db2 on Cloud configuration details from an example configuration: 
 
+**Example: Credential**
+``` json
+{
+  "hostname": "dashdb-enterprise-xxxxxxx.services.dal.bluemix.net",
+  "password": "my-key",
+  "https_url": "https://dashdb-enterprise-xxxxxxx.services.dal.bluemix.net",
+  "port": 50000,
+  "ssldsn": "DATABASE=BLUDB;HOSTNAME=dashdb-enterprise-xxxxxx.services.dal.bluemix.net;PORT=50001;PROTOCOL=TCPIP;UID=bluadmin;PWD=my-key;Security=SSL;",
+  "host": "dashdb-enterprise-xxxxxxx.services.dal.bluemix.net",
+  "jdbcurl": "jdbc:db2://dashdb-enterprise-xxxxxxx.services.dal.bluemix.net:50000/BLUDB",
+  "uri": "db2://bluadmin:my-key@dashdb-enterprise-xxxxxxx.services.dal.bluemix.net:50000/BLUDB",
+  "db": "BLUDB",
+  "dsn": "DATABASE=BLUDB;HOSTNAME=dashdb-enterprise-xxxxxxx.services.dal.bluemix.net;PORT=50000;PROTOCOL=TCPIP;UID=bluadmin;PWD=my-key;",
+  "username": "bluadmin",
+  "ssljdbcurl": "jdbc:db2://dashdb-enterprise-xxxxxxx.services.dal.bluemix.net:50001/BLUDB:sslConnection=true;"
+}
+```
+
+Use the following table to populate your datasource. 
+
+| IBM FHIR Server Configuration  | Description |
+|----|----|
+| serverName | from the credential object select `hostname` |
+| portNumber | from the credential object select  `port` |
+| databaseName | from the credential object select  `db`, generally always `BLUDB` |
+| apiKey | from the `password` field |
```

Comment:
I don't think this is right.  Instead, it should be the API Key generated for the user created in the FHIRUSER group (i.e. the IBMid user created from https://github.com/IBM/FHIR/blob/master/docs/src/pages/guides/DB2OnCloudSetup.md#fhirserver-user-and-api-key )

Reply:
The new language looks good, thanks.